lib/gc-dialects/OneDNNGraph/OneDNNGraphOps.cpp

```diff
@@ -0,0 +1,184 @@
+//===- OneDNNGraphOps.h - OneDNN input dialect ops --------------*- C++ -*-===//
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "gc-dialects/OneDNNGraph/OneDNNGraphOps.h"
+#include "gc-dialects/OneDNNGraph/OneDNNGraphDialect.h"
+#include "mlir/IR/OpImplementation.h"
+#include "llvm/Support/Debug.h"
+
+#define GET_OP_CLASSES
+#include "gc-dialects/OneDNNGraph/OneDNNGraphOps.cpp.inc"
+
+namespace mlir {
+namespace onednn_graph {
+
+// https://github.com/onnx/onnx/blob/main/docs/Broadcasting.md
```

Comment:
What is this reference for and how it is related?

Reply:
This is before we switched to `ResultsBroadcastableShape`, kept it so @BRUCE11111 can migrate his PR to here. inferBroadcastShape will be deleted.